src/components/calcite-split-button/calcite-split-button.tsx

```diff
@@ -21,6 +21,12 @@ export class CalciteSplitButton {
   /** is the control disabled  */
   @Prop({ reflect: true }) disabled?: boolean;
 
+  /**
+   * Is the dropdown currently active or not
+   * @internal
```

Comment:
I can make this public in a separate PR or we could do it at a later time.

Reply:
👍  to revisit making it public later.